curiouslearning/FeedTheMonsterJS
# Feat: ensures ensures that load Title and Feedback Custom Font loads (FM-278)

# Changes
- Making sure that the title and feedback custom font loads before executing other code

# How to test
- Turn off WIFI
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- In game screen play and give correct answer
- Feedback text font should display properly


Ref: [FM-278](https://curiouslearning.atlassian.net/browse/FM-278)


[FM-278]: https://curiouslearning.atlassian.net/browse/FM-278?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
